gevent build failure - missing rename

### Setup
_Package Name:_ `gevent`
_Package Version:_ n/a

_NAS Model:_ n/a
_NAS Architecture:_ n/a
_DSM version:_ n/a

### Expected behavior
While trying to figure out build failures against `ffsync` I ended-up noticing a `gevent` build issue due to missing `rename` command line utility.  Needs to be included in both the default Docker image and `README.rst`

### Log output
```
removing build/bdist.linux-x86_64/wheel
/bin/sh: 1: rename: not found
Makefile:20: recipe for target 'myInstall' failed
make[2]: *** [myInstall] Error 127
make[2]: Leaving directory '/home/spksrc/git-cmake2/spksrc/cross/gevent'
../../mk/spksrc.depend.mk:44: recipe for target 'depend_target' failed
make[1]: *** [depend_target] Error 2
make[1]: Leaving directory '/home/spksrc/git-cmake2/spksrc/spk/ffsync'
../../mk/spksrc.spk.mk:434: recipe for target 'arch-apollolake' failed
make: [arch-apollolake] Error 2 (ignored)
```

--- a/.dockerignore
+++ b/.dockerignore
@@ -1,10 +1,2 @@
-distrib
-packages
-native/*/work*
-cross/*/work*
-kernel/*/work*
-spk/*/work*
-toolchains/*/work*
-local.mk
-*.DS_Store
-*~
+# exclude all files in the repository
+*

--- a/Dockerfile
+++ b/Dockerfile
@@ -6,54 +6,58 @@ ENV LANG C.UTF-8
 # Manage i386 arch
 RUN dpkg --add-architecture i386
 
-# Install required packages (in sync with README.rst instructions
+# Install required packages (in sync with README.rst instructions)
 RUN apt-get update && apt-get install --no-install-recommends -y \
-        autogen \
-        automake \
-        bc \
-        bison \
-        build-essential \
-        check \
-        cmake \
-        curl \
-        cython \
-        debootstrap \
-        expect \
-        flex \
-        g++-multilib \
-        gettext \
-        git \
-        gperf \
-        imagemagick \
-        intltool \
-        libbz2-dev \
-        libc6-i386 \
-        libcppunit-dev \
-        libffi-dev \
-        libgc-dev \
-        libgmp3-dev \
-        libltdl-dev \
-        libmount-dev \
-        libncurses-dev \
-        libpcre3-dev \
-        libssl-dev \
-        libtool \
-        libunistring-dev \
-        lzip \
-        mercurial \
-        ncurses-dev \
-        php \
-        pkg-config \
-        python3 \
-        python3-distutils \
-        scons \
-        subversion \
-        swig \
-        unzip \
-        xmlto \
-        zlib1g-dev && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+		autogen \
+		automake \
+		bc \
+		bison \
+		build-essential \
+		check \
+		cmake \
+		curl \
+		cython \
+		debootstrap \
+		ed \
+		expect \
+		flex \
+		g++-multilib \
+		gawk \
+		gettext \
+		git \
+		gperf \
+		imagemagick \
+		intltool \
+		libbz2-dev \
+		libc6-i386 \
+		libcppunit-dev \
+		libffi-dev \
+		libgc-dev \
+		libgmp3-dev \
+		libltdl-dev \
+		libmount-dev \
+		libncurses-dev \
+		libpcre3-dev \
+		libssl-dev \
+		libtool \
+		libunistring-dev \
+		lzip \
+		mercurial \
+		ncurses-dev \
+		php \
+		pkg-config \
+		pngquant \
+		python3 python3-distutils \
+		rename \
+		scons \
+		subversion \
+		swig \
+		texinfo \
+		unzip \
+		xmlto \
+		zlib1g-dev && \
+	apt-get clean && \
+	rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
 # Install setuptools, wheel and pip for Python3
 RUN wget https://bootstrap.pypa.io/get-pip.py -O - | python3

--- a/Dockerfile
+++ b/Dockerfile
@@ -46,8 +46,8 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
 		ncurses-dev \
 		php \
 		pkg-config \
-		pngquant \
-		python3 python3-distutils \
+		python3 \
+		python3-distutils \
 		rename \
 		scons \
 		subversion \

--- a/README.rst
+++ b/README.rst
@@ -26,7 +26,7 @@ A virtual machine based on an 64-bit version of Debian 10 stable OS is recommend
 * Install the requirements (in sync with Dockerfile)::
 
     sudo dpkg --add-architecture i386 && sudo apt-get update
-    sudo apt install autogen automake bc bison build-essential check cmake curl cython debootstrap expect flex g++-multilib gettext git gperf imagemagick intltool libbz2-dev libc6-i386 libcppunit-dev libffi-dev libgc-dev libgmp3-dev libltdl-dev libmount-dev libncurses-dev libpcre3-dev libssl-dev libtool libunistring-dev lzip mercurial ncurses-dev php pkg-config python3 python3-distutils scons subversion swig unzip xmlto zlib1g-dev
+    sudo apt install autogen automake bc bison build-essential check cmake curl cython debootstrap ed expect flex g++-multilib gawk gettext git gperf imagemagick intltool libbz2-dev libc6-i386 libcppunit-dev libffi-dev libgc-dev libgmp3-dev libltdl-dev libmount-dev libncurses-dev libpcre3-dev libssl-dev libtool libunistring-dev lzip mercurial ncurses-dev php pkg-config python3 python3-distutils rename scons subversion swig texinfo unzip xmlto zlib1g-dev
     sudo pip install -U setuptools pip wheel httpie
 
 * You may need to install some packages from testing like autoconf. Read about Apt-Pinning to know how to do that.

